inf-clojure.el

```diff
@@ -932,7 +1070,7 @@ Indent FORM.  FORM is expected to have been trimmed."
   (let ((clojure-process (inf-clojure-proc)))
     ;; ensure the repl buffer scrolls. See similar fix in CIDER:
     ;; https://github.com/clojure-emacs/cider/pull/2590
-    (with-selected-window (or (get-buffer-window inf-clojure-buffer)
+    (with-selected-window (or (get-buffer-window (inf-clojure-buffer))
```

Comment:
You are invoking `inf-clojure-buffer`. But that still seems to be a `defvar` in this diff. I don't see why this would be a function.

I suspect it's meant to grab either the regular buffer or the socket buffer. If that's so, what do you do if both are set? Not sure if the regular buffer is cleared when you start a socket repl. But if you choose a different repl with `inf-clojure-set-repl`, can you swap back to a socket repl? Seems like nothing knows to set the socket repl buffer except for the code to actually start the process.

Reply:
> the regular buffer or the socket buffer.

Socket buffer as in the server process buffer? (the REPL buffer is the connection process already)